arch/rv32i/src/pmp.rs

```diff
@@ -478,148 +492,161 @@ impl kernel::mpu::MPU for PMPConfig {
+                            }
+                            5 => {
+                                // Disable access up to the start address
+                                csr::CSR.pmpcfg2.modify(
+                                    csr::pmpconfig::pmpcfg::r3::CLEAR
+                                        + csr::pmpconfig::pmpcfg::w3::CLEAR
+                                        + csr::pmpconfig::pmpcfg::x3::CLEAR
+                                        + csr::pmpconfig::pmpcfg::a3::TOR,
+                                );
+                                csr::CSR.pmpaddr10.set((start as u32) >> 2);
+
+                                // Set access to end address
+                                csr::CSR.pmpcfg2.set(cfg_val << 24 | csr::CSR.pmpcfg0.get());
+                                csr::CSR.pmpaddr11.set((start as u32 + size as u32) >> 2);
+                            }
+                            6 => {
+                                // Disable access up to the start address
+                                csr::CSR.pmpcfg3.modify(
+                                    csr::pmpconfig::pmpcfg::r0::CLEAR
+                                        + csr::pmpconfig::pmpcfg::w0::CLEAR
+                                        + csr::pmpconfig::pmpcfg::x0::CLEAR
+                                        + csr::pmpconfig::pmpcfg::a0::TOR,
+                                );
+                                csr::CSR.pmpaddr12.set((start as u32) >> 2);
+
+                                // Set access to end address
+                                csr::CSR.pmpcfg3.set(cfg_val << 8 | csr::CSR.pmpcfg0.get());
+                                csr::CSR.pmpaddr13.set((start as u32 + size as u32) >> 2);
+                            }
+                            7 => {
+                                // Disable access up to the start address
+                                csr::CSR.pmpcfg3.modify(
+                                    csr::pmpconfig::pmpcfg::r3::CLEAR
+                                        + csr::pmpconfig::pmpcfg::w3::CLEAR
+                                        + csr::pmpconfig::pmpcfg::x3::CLEAR
+                                        + csr::pmpconfig::pmpcfg::a3::TOR,
+                                );
+                                csr::CSR.pmpaddr14.set((start as u32) >> 2);
+
+                                // Set access to end address
+                                csr::CSR.pmpcfg3.set(cfg_val << 24 | csr::CSR.pmpcfg0.get());
+                                csr::CSR.pmpaddr15.set((start as u32 + size as u32) >> 2);
+                            }
+                            _ => break,
                         }
-                        _ => break,
                     }
-                }
-                None => {}
-            };
-        }
+                    None => {}
+                };
+            }
+            // Clear the dirty flag
```

Comment:
Non informative comment, it is obvious from the code that the dirty flag is cleared.

Reply:
Done.